packages/utils/package.json

```diff
@@ -68,18 +68,15 @@
   },
   "dependencies": {
     "@eslint-community/eslint-utils": "^4.4.0",
-    "@types/json-schema": "^7.0.15",
```

Comment:
It was forked in https://github.com/typescript-eslint/typescript-eslint/pull/6963, so now it's unused